parseIntTransacAmounts.js

```diff
@@ -0,0 +1,9 @@
+function parseIntTransacAmounts(allTransactions) {
+    for (let i=0;i<allTransactions.length;i++) {

```

Comment:
This is a little nicer if you use a `forEach`:

    allTransactions.forEach(transaction => {
      transaction.amountPounds = parseInt(transaction.amountPounds);
      transaction.amountPennies = parseInt(transaction.amountPennies);
    })

(Also, more spaces make it a touch easier to read.)